### DISCUSSION TITLE
Question about MessagePassing network's GCN implementation

### DISCUSSION THREAD

**Discussion by hydra324** [pyg-team/pytorch_geometric#6380] [Q&A] [answered]
The normalization step in GCN implementation using message passing base class uses `row,col = edge_index` in the documentation. What exactly does this do?

**Comment by rusty1s** [comment, marked as answer]
`edge_index` denotes a COO tensor, i.e. it holds pairs of source and destination node indices. `row` refers to all the source node indices, and `col` refers to all the destination node indices.

**Reply by hydra324** [reply]
So, I think for an undirected graph, it shouldn't matter whether I compute degree using row or col. Thank you.
